web/src/Toast.jsx

```
@@ -0,0 +1,121 @@
+  return (
+    <div
+      className={`toast-item toast-${type} ${isExiting ? 'toast-exit' : 'toast-enter'}`}
+      role="alert"
+    >
+      <div className={`toast-icon ${type === 'loading' ? 'toast-spinner' : ''}`}>
+        {iconMap[type] || 'i'}
+      </div>
+      <div className="toast-content">{message}</div>
+      {type !== 'loading' && (
+        <button className="toast-close" onClick={handleDismiss} aria-label="Dismiss">
+          ×
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)